setup.py

```diff
@@ -53,6 +53,7 @@ def find_scripts(script_dir, prefix):
         'numba',
         'numpy',
         'pandas',
+        'pyirf~=0.2.0',
```

Comment:
We just released 0.3

Reply:
Are the things introduced previously here compatible?

Reply:
There is already pyirf stuff in the master?

Reply:
I mean I just wanted to solve the install issue but yeah I can bump to 0.3.0 at the same time

Reply:
Ah, yes the reading part. I had a quick glimpse and I think nothing with respect to that changed in 0.3